dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -61,6 +61,7 @@
 import joptsimple.OptionSpec;
 import joptsimple.ValueConversionException;
 import joptsimple.ValueConverter;
+import org.apache.arrow.util.Preconditions;
```

Comment:
I think this meant to use Guava's preconditions.

Reply:
Fixed.